posthog/migrations/0299_add_constraints_to_person_override.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 3.2.16 on 2023-02-02 14:32
+
+import django.db.models.expressions
+from django.db import migrations, models
+
+from posthog.models.person import PersonOverride
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0298_add_insight_queries"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            f"""
```

Comment:
Can we make sure this one and the one in setup_test_environment are always in sync somehow?

Reply:
Yeah, we can put the SQL code next to the model and then just import it in both places.